scripts/hail_batch/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -185,11 +150,83 @@ def function_that_merges_dataframes(*df_list):
     return merged_df.to_string()
 
 
-merge_job = b.new_python_job(name='merge_scatters')
-result_second = merge_job.call(
-    function_that_merges_dataframes, *spearman_dfs_from_scatter
+# Create click command line to enter dependency files
+@click.command()
+@click.option(
+    '--expression', required=True, help='A sample x gene TSV of expression values'
+)
+@click.option('--genotype', required=True, help='A TSV of genotypes for each sample')
```

Comment:
Is there a way to make some of the input files (such as the genotype file) go over all chromosomes rather than having the user input the file?